src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -59,14 +57,14 @@ public class SchemaBuilder
         /// <returns>Created schema.</returns>
         public virtual Schema Build(string typeDefinitions)
         {
-            var document = Parser.Parse(typeDefinitions, new ParserOptions { Ignore = IgnoreComments ? IgnoreOptions.IgnoreComments : IgnoreOptions.None });
+            var document = Parser.Parse(typeDefinitions, new ParserOptions { Ignore = IgnoreComments ? IgnoreOptions.Comments : IgnoreOptions.None });
             Validate(document);
             return BuildSchemaFrom(document);
         }
 
         protected virtual void Validate(GraphQLDocument document)
         {
-            var definitionsByName = document.Definitions.OfType<GraphQLTypeDefinition>().Where(def => !(def is GraphQLTypeExtensionDefinition)).ToLookup(def => def.Name!.Value);
```

Comment:
`GraphQLTypeExtensionDefinition` does not inherit from `GraphQLTypeDefinition` anymore.